src/main/java/com/example/hop_oasis/controller/CartController.java

```diff
@@ -1,93 +1,51 @@
 @RequestMapping("/cart")
-@SessionAttributes("cart")
+@RequiredArgsConstructor
+@Log4j2
 public class CartController {
-    private final CartService cartService;
-    @ModelAttribute("cart")
-    public List<CartItemDto> createCartItems() {
-        return cartService.createCartItems();
-    }
-    @GetMapping("/session-id")
-    public ResponseEntity<String>getSessionId(HttpSession session, HttpServletResponse response) {
-        String sessionId = session.getId();
-        Cookie cookie = new Cookie("SESSIONID", sessionId);
-        cookie.setHttpOnly(true);
-        cookie.setPath("/");
-        cookie.setMaxAge(7 * 24 * 60 * 60);
-        cookie.setSecure(true);
-
-        response.addCookie(cookie);
 
-        response.setHeader("Set-Cookie", String.format("SESSIONID=%s; HttpOnly; Path=/; Max-Age=%d; Secure; SameSite=None",
-                sessionId, 7 * 24 * 60 * 60));
-
-        return ResponseEntity.ok().body(sessionId);
-    }
-    @GetMapping("/session-id-from-cookie")
-    public ResponseEntity<String> getSessionIdFromCookie(HttpServletRequest request) {
-        Cookie[] cookies = request.getCookies();
-        if (cookies != null) {
-            for (Cookie cookie : cookies) {
-                if ("SESSIONID".equals(cookie.getName())) {
-                    return ResponseEntity.ok().body("Session ID from cookie: " + cookie.getValue());
-                }
-            }
-        }
-        return ResponseEntity.ok().body("Session ID not found in cookies");
-    }
+    private final CartService cartService;
 
     @GetMapping
     public ResponseEntity<CartDto> find() {
-        return ResponseEntity.ok().body(cartService.getAllItems(createCartItems()));
+        return ResponseEntity.ok().body(cartService.getAllItems());
     }
 
-//    @PostMapping("/items")
-//    public ResponseEntity<CartItemDto> add(@RequestBody ItemRequestDto itemRequest) {
-//        CartItemDto dto = cartService.add(itemRequest.getItemId(), itemRequest.getQuantity(), itemRequest.getItemType());
-//        return ResponseEntity.ok(dto);
-//    }
-    @GetMapping("/items")public ResponseEntity<CartItemDto> addItem(
-            @RequestParam("itemId") Long itemId,
-            @RequestParam("quantity") int quantity,
-            @RequestParam("itemType") ItemType itemType) {
-        CartItemDto dto = cartService.add(itemId, quantity, itemType);
+    @PostMapping("/add")
+    public ResponseEntity<CartItemDto> addItem(@RequestBody ItemRequestDto itemRequestDto) {
+        CartItemDto dto = cartService.add(itemRequestDto.getItemId(), itemRequestDto.getQuantity(), itemRequestDto.getItemType());
         return ResponseEntity.ok(dto);
     }
 
-    @PutMapping
-    public ResponseEntity<CartItemDto> updateQuantity(@RequestBody ItemRequestDto itemRequest) {
-        CartItemDto dto = cartService.updateQuantity(itemRequest.getItemId(), itemRequest.getQuantity(), itemRequest.getItemType());
-        return ResponseEntity.ok(dto);
+    @PutMapping("/update")
```

Comment:
`/update` is redundant. PUT method by itself means that the resource is updated

Reply:
done